i18n/en.json

```diff
@@ -1815,6 +1815,53 @@
+  "admin.reporting.workspace_optimization.data_privacy.description": "Get better insight and control over your data.",
+  "admin.reporting.workspace_optimization.data_privacy.retention.description": "A lot of organizations in highly regulated indsutries require more control and insight with thier data. Become more aware and take control of your data by trying out data retention and compliance features.",
+  "admin.reporting.workspace_optimization.data_privacy.retention.title": "Become more data aware",
+  "admin.reporting.workspace_optimization.data_privacy.title": "Data Privacy",
+  "admin.reporting.workspace_optimization.downloadUpdate": "Download Update",
+  "admin.reporting.workspace_optimization.ease_of_management.description": "We have suggestions that could make your managemenet easier.",
+  "admin.reporting.workspace_optimization.ease_of_management.guests_accounts.description": "We noticed several accounts using different domains from your Site URL. Gain more control over what other organizations can access with the guest account feature.",
+  "admin.reporting.workspace_optimization.ease_of_management.guests_accounts.title": "Guest Accounts recommended",
+  "admin.reporting.workspace_optimization.ease_of_management.ldap.description": "You’ve reached over 100 users, we can reduce your manual management pains through AD/LDAP with features like easier onboarding, automatic deactivations and automatic role assignments.",
+  "admin.reporting.workspace_optimization.ease_of_management.ldap.title": "AD/LDAP integration recommended",
+  "admin.reporting.workspace_optimization.ease_of_management.title": "Ease of management",
+  "admin.reporting.workspace_optimization.overall_workspace_score": "Overall Workspace Score",
+  "admin.reporting.workspace_optimization.overall_workspace_score_description": "Stay on top of how optimized your workspace is! This score is a total of the categories of key indicators of health and growth below. Ensure your workspace is in its most optimized state so you and your user base get the most out of Mattermost.",
+  "admin.reporting.workspace_optimization.performance.description": "Your server could use some performance tweaks.",
+  "admin.reporting.workspace_optimization.performance.search.description": "Your server has reached over 500 users and 2 million posts which could result in slow search performance. We recommend starting an enterprise trial with the elastic search feature for better performance.",
+  "admin.reporting.workspace_optimization.performance.search.title": "Search performance",
+  "admin.reporting.workspace_optimization.performance.title": "Performance",
+  "admin.reporting.workspace_optimization.security.description": "There are security concerns you should look at.",
+  "admin.reporting.workspace_optimization.security.login_attempts.description": "{attempts} Failed login attempts have been detected. We recommend looking at the security logs to understand the risk.",
```

Comment:
```suggestion
  "admin.reporting.workspace_optimization.security.login_attempts.description": "{attempts} failed login attempts have been detected on this workspace. We recommend reviewing your [security logs] to understand this security risk.",
```

Reply:
Question: How would an admin review security logs in MM? (not sure what docs link to recommend yet)

Reply:
afaik we only have server logs that are more general and are not tailored to security. right @mkraft ?

Reply:
yeah this should be reworded to say "server logs"